pep-3156.txt

```diff
@@ -1647,15 +1720,31 @@ callback-based frameworks like Twisted.  After converting a coroutine
 into a Task, callbacks can be added to the Task.
 
 To convert a coroutine into a task, call the coroutine function and
-pass the resulting coroutine object to the ``asyncio.Task()``
-constructor.  You may also use ``asyncio.async()`` for this purpose.
+pass the resulting coroutine object to the ``loop.create_task()``
+method.  You may also use ``asyncio.ensure_future()`` for this purpose.
 
 You may ask, why not automatically convert all coroutines to Tasks?
 The ``@asyncio.coroutine`` decorator could do this.  However, this would
 slow things down considerably in the case where one coroutine calls
 another (and so on), as switching to a "bare" coroutine has much less
 overhead than switching to a Task.
 
+A Task is inherited from ``Future`` but adds new methods:
+
+- ``current_task(loop=None)``. A *class method* returning the
```

Comment:
Why are these class methods rather than static methods (or even better, methods of `loop`)? They are accessed as `Task.current_task()`, so subclasses of `Task` have no meaningful way to override them. 

How exactly does a `Task` get into `current_task()`? Don't we need to standardize a `set_current_task()` method for alternative coroutine runners?